run_project_tests.py

```diff
@@ -160,17 +206,8 @@ def setup_commands(optbackend):
     compile_commands, clean_commands, test_commands, install_commands, \
         uninstall_commands = get_backend_commands(backend, do_debug)
 
-def get_relative_files_list_from_dir(fromdir: Path) -> T.List[Path]:
-    return [file.relative_to(fromdir) for file in fromdir.rglob('*') if file.is_file()]
-
-def platform_fix_name(fname: str, compiler, env) -> str:
-    # canonicalize compiler
-    if (compiler in {'clang-cl', 'intel-cl'} or
-       (env.machines.host.is_windows() and compiler == 'pgi')):
-        canonical_compiler = 'msvc'
-    else:
-        canonical_compiler = compiler
-
+# TODO try to eliminate or at least reduce this function
```

Comment:
Yeah, it's horrible :) I think more could be removed by adding 'lib' and 'so' installed files types, but that can certainly be done as a follow-up.

Reply:
Yeah, I don't want to do this in this PR. I might look into this later.